GameBoard/Models/FriendsSidebar/FriendsSidebarViewModel.cs

```diff
@@ -1,27 +1,43 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
-using GameBoard.Models.User;
+using GameBoard.Models.Groups;
 
 namespace GameBoard.Models.FriendsSidebar
 {
     public class FriendsSidebarViewModel
     {
-        public IEnumerable<UserViewModel> Friends { get; }
+        public IEnumerable<GroupViewModel> Groups { get; }
 
         public bool Toggled { get; }
 
+        public bool GroupInviteEnabled { get; }
```

Comment:
When is it disabled?

Reply:
It's a bool that is checked for when deciding if there should be a group invite button on the sidebar. The default sidebar model is the most basic one, so there is no invite button and thus the bool is false

Reply:
Resolved